src/test/java/org/opensearch/security/dlic/rest/api/AbstractRestApiUnitTest.java

```diff
@@ -195,11 +195,11 @@ protected void checkGeneralAccess(int status, String username, String password)
 		rh.sendAdminCertificate = sendAdminCertificate;
 	}
 
-	protected String checkReadAccess(int status, String username, String password, String indexName, String type,
+	protected String checkReadAccess(int status, String username, String password, String indexName, String actionType,
```

Comment:
Fixing the naming here as OpenSearch 2.0+ supports actionType in the API path and dropped the support for type.